app/views/my_modules/archive.html.erb

```diff
@@ -1,4 +1,4 @@
-<% provide(:head_title, raw(t("my_modules.module_archive.head_title", project: @my_module.experiment.project.name, module: @my_module.name))) %>
+<% provide(:head_title, sanitize_input(t("my_modules.module_archive.head_title", project: @my_module.experiment.project.name, module: @my_module.name))) %>
```

Comment:
Same here: escape_input(@my_module.experiment.project.name) and escape_input(@my_module.name)